tests/plugins/test_plugins_manager.py

```diff
@@ -350,6 +357,21 @@ class MacroPlugin(AirflowPlugin):
             # rendering templates.
             assert hasattr(macros, MacroPlugin.name)
 
+    def test_registering_plugin_listeners(self):
+        from airflow import plugins_manager
+
+        with mock.patch('airflow.plugins_manager.plugins', []):
+            plugins_manager.load_plugins_from_plugin_directory()
+            plugins_manager.integrate_listener_plugins()
+
+            assert get_listener_manager().has_listeners()
+            assert get_listener_manager().pm.get_plugins().pop().__class__.__name__ == "PluginListener"
+
+    @pytest.fixture(autouse=True)
+    def clear_listeners(self):
+        yield
+        get_listener_manager().clear()
```

Comment:
```suggestion
```

Already have this as a module level fixture.

Reply:
Removed.